lib/oli_web/live/admin/admin_view.ex

```diff
@@ -60,6 +60,11 @@ defmodule OliWeb.Admin.AdminView do
                 Manage LTI 1.3 Registrations
               </a>
             </li>
+            <li>
```

Comment:
This is in the wrong place, this "Manage VR User Agents" should be in the "System Admin" portion of this view.

Reply:
On it!